run.py

```diff
@@ -621,7 +638,10 @@ def default_args(self):
         # Find rules.
         rules = glob.glob(os.path.join(self.directory, "*.rules"))
         if not rules:
-            args += ["-S", "/dev/null"]
+            if sys.platform == "win32":
+                args += ["--disable-detection"]
+            else:
+                args += ["-S", "/dev/null"]
```

Comment:
Any reason not to just use --disable-detection for both cases here?

Reply:
We could do that. The code paths are different though, so I guess it makes some sense to test both. In the past we've had large perf differences between the opts.

Reply:
I guess for the purpose of suricata-verify we can make this change indeed.

Reply:
See #27